doc/src/records/rsys.md

```diff
@@ -498,13 +498,13 @@ Coefficient of performance at outdoor dry-bulb temperature of 5 ^o^F, minimum sp
 <%= member_table(
   units: "",
   legal_range: "Calculated from rsCAP05, rsCOP05",
-  default: "0.0",
+  default: "rsCOP05",
   required: "No",
   variability: "Before set up or at the end of interval") %>
 
 **rsloadFMin115=*float***
 
-Fraction of a clg minimum load at 115 ^o^F.
+Fraction of a cooling minimum load at 115 ^o^F (cycle below this).
```

Comment:
It feels like you're taking a guess at this from the somewhat cryptic description in CNRECS. It's a good example of when you should reach out with a quick question on slack to clarify things. In the case of all the `rsloadFMin*` inputs, they are all the fraction of the minimum capacity to the maximum capacity at stated temperature.

@chipbarnaby, I also think the word "load" is confusing, since it is really more about capacity.

Reply:
I agree load is not a good term.  I thought about renaming the input names but did not do it.  But certainly the documentation should be clear.  Something like --

Ratio of total cooling capacity at minimum (non-cycling) speed to full-speed total cooling capacity at 115 F outdoor dry bulb temperature.